Tests/MQTTnet.Core.Tests/MQTTv5/Client_Tests.cs

```diff
@@ -12,12 +15,51 @@
 using MQTTnet.Protocol;
 using MQTTnet.Server;
 using MQTTnet.Tests.Mockups;
+using MQTTnet.Tests.MQTTv5.ExtendedAuth;
 
 namespace MQTTnet.Tests.MQTTv5
 {
     [TestClass]
     public class Client_Tests
     {
+        [TestMethod]
```

Comment:
Please have a look at the "TestEnvironment" class and check if you can make use of that for testing. It already wraps ports and also does a lot of validations and error checking.